Base/res/js/Spreadsheet/runtime.js

```diff
@@ -1,8 +1,79 @@
+// FIXME: Figure out a way to document non-function entities too.
+class Position {
+    constructor(column, row, sheet) {
+        this.column = column;
+        this.row = row;
+        this.sheet = sheet ?? thisSheet;
+        this.name = column + "" + row;
```

Comment:
I prefer this, but it's up to you :)

```suggestion
        this.name = `${column}${row}`;
```